src/Microsoft.DocAsCode.MarkdigEngine/MarkdownExtensions.cs

```diff
@@ -47,7 +45,9 @@ public static MarkdownPipelineBuilder UseMarkdigAdvancedExtensions(this Markdown
                 .UseLineNumber(context, parameters)
                 .UseMonikerRange()
                 .UseValidators(context, parameters)
-                .UseInteractiveCode();
+                .UseInteractiveCode()
+                .UseRow()
+                .UseNestedColumn();
```

Comment:
is it better to merge these two extensions into one? or separate into two extensions is by design?

Reply:
For now, we're going to keep the functionality separate.